config/webpack/config.base.js

```diff
@@ -84,7 +85,7 @@ module.exports = ({ sourceDir, distDir }) => ({
   },
   output: {
     path: distDir,
-    publicPath: "/",
+    publicPath: STATIC_URL,
```

Comment:
This line adds public paths so you don't have to do this for every asset (images, fonts, etc)